stubs/MarkupSafe/markupsafe/__init__.pyi

```diff
@@ -10,12 +11,12 @@ class Markup(text_type):
     def __html__(self) -> Markup: ...
     def __add__(self, other: text_type) -> Markup: ...
     def __radd__(self, other: text_type) -> Markup: ...
-    def __mul__(self, num: int) -> Markup: ...
-    def __rmul__(self, num: int) -> Markup: ...
+    def __mul__(self, num: int) -> Markup: ...  # type: ignore
+    def __rmul__(self, num: int) -> Markup: ...  # type: ignore
```

Comment:
`__mul__` explicitly checks for an `int` instance:
https://github.com/pallets/markupsafe/blob/ca3ecf6d1a981c5b5316e777bb0b7d8fc3ebbaf3/src/markupsafe/__init__.py#L95-L99